workspaces/adventure-pack/goodies/python3/src/flatten/test.py

```diff
@@ -0,0 +1,19 @@
+from . import *
+
+
+def test_flatten():
+    l = [[0, 1, 2, 3], [[4, 5, 6], [7, 8, 9]], 10, 11, 12]
+    flattened = list(flatten(l))
+    assert isinstance(l[0], list)
```

Comment:
What's the intent behind this assertion?

Reply:
While I was still writing/debugging the code, I couldn't print() stuff so I just asserted it to check if the value was true or false and then I forgot to remove it.